AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -574,6 +574,32 @@ -(void) closeWindow
     [self.currentWindow performAction:@"AXCancel"];
 }
 
+- (NSString *)executeShellScript:(NSString *)script
+{
+    NSTask *task = [[NSTask alloc] init];
```

Comment:
I assume NSTask starts with an empty environment by default. Shouldn't we inherit the current system env by default?